infrastructure/prime-app-template.yml

```diff
@@ -371,7 +387,7 @@ objects:
                   secretKeyRef:
                     name: ${SVC_NAME}-postgres
                     key: database-password
-              - name: POSTGRESQL_USERNAME
+              - name: POSTGRESQL_USER
```

Comment:
Does this secret exist already? Do we have to time the deployments to prod?

Reply:
There was a change to the Entrypoint. Along with that, POSTGRESQL_USERNAME got added with POSTGRESQL_USER (duplicate) before. this is a fix for that.
Won't have to time the deploy.